src/main/java/com/pscheduler/util/GradeBuilderFactory.java

```diff
@@ -0,0 +1,141 @@
+package com.pscheduler.util;
+
+import com.pscheduler.server.model.CourseGPA;
+
+public class GradeBuilderFactory {
```

Comment:
Technically just a builder right now. If we get to the serverless part might need to add the factory part.

Reply:
Renamed to just 'GradeBuilder' now.